### ISSUE TITLE
Some operators in Python don't compile

### ISSUE THREAD

**Issue by tbennun** [spcl/dace#38]
**Describe the bug**
Adding operators to element-wise statements, or augmented assignment, fails to run through the frontend.

**To Reproduce**
Run the program below:
```python
@dace.program
def transpose(A: dace.float32[M, K], B: dace.float32[K, M]):
    for i, j in dace.map[0:M, 0:K]:
        B[j, i] = A[i, j] + 1
```

**Comment by tbennun** [comment]
Probably a missed corner case in `InlineSDFG`

**Comment by tbennun** [comment]
Fixed in #44.
